integration-libs/punchout/root/model/punchout.model.ts

```diff
@@ -6,9 +6,9 @@
 
 export const PUNCHOUT_SESSION_KEY = 'sid';
 export const PUNCHOUT_SESSION_ID = 'punchoutSessionId';
-export const PUNCHOUT_SESSION_PAGE_URL = '/punchout/cxml/session';
```

Comment:
make sure it does not break the navigation guard when we compare the path

Reply:
all good, it checks do wider string includes thin